src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/SqlCommand.cs

```diff
@@ -730,7 +730,7 @@ override public string CommandText
             {
                 SqlClientEventSource.Log.TryTraceEvent("<sc.SqlCommand.set_CommandText|API> {0}, String Value = '{1}'", ObjectID, value);
 
-                if (0 != ADP.SrcCompare(_commandText, value))
+                if (_commandText != value)
```

Comment:
This seems unrelated?

Reply:
Sort of. The intent was to clean up ADP as well but the PR was getting too large and complicated so I stopped part way. This change on it's own isn't harmful though so I left it in. 

Functions like SrcCompare and DestCompare which just do string comparisons don't really have any benefit to being functions instead of inline, they're trivial and not worth the extra effort required to investigate what they're doing.